app/Http/Controllers/NewOrderController.php

```diff
@@ -189,4 +190,17 @@ public function chargeCustomerOrder(Request $request, DirectDebitService $direct
         }
         return $this->sendSuccess([], 'Customer debited successfully and amortization(s) has been updated');
     }
+
+    public function changeOrderStatus(Request $request)
+    {
+       $validated =  $this->validate($request, [
+            'status' => ['required', 'string', Rule::in(['Approved', 'Pending', 'Active', 'Closed', 'Repossessed', "Completed"])],
```

Comment:
@darmhoo Let us use the order status id 

Create an api for the frontend to get order statuses available 

Let the frontend send the order status id instead of the name of status.

We can use the existing rule to check if an order status with that id exists and it also gives us the flexibility of not needing to worry of the name change of status in DB and also the possibility of adding more status and not needing to come alter the code